src/Routes/Filter/RouteFilterOptions.php

```diff
@@ -78,7 +75,11 @@ public function toArray(): array
     {
         $values = [];
         foreach (get_object_vars($this) as $key => $value) {
-            if (! empty($value)) {
+            if (in_array($key, $this->getFilterOptionsMinusMethods()) && $value !== "") {
```

Comment:
The entire `toArray()` method is unused in the command AFAICT. Can it be dropped?